### DISCUSSION TITLE
Wave Application Stops Working Abruptly

### DISCUSSION THREAD

**Discussion by nivzx** [h2oai/wave#2316] [Q&A] [answered]
Hello,
I'm using Wave v1.0.0 in an app I'm working on in which I get an error where the app stops working abruptly with following instance logs and browser error:
```
{"level":"error","cmd":"/resources/venv/bin/uvicorn","process":99,"time":"2024-04-16T04:31:19Z","message":"Cmd terminated, killing all the other cmds"}
{"level":"error","cmd":"/resources/venv/waved","process":91,"error":"signal: killed","time":"2024-04-16T04:31:19Z","message":"Cmd terminated, killing all the other cmds"}
failed: failed running wave app: app error: cmd interrupted
```

<img width="1720" alt="Screenshot 2024-04-16 at 10 40 10" src="https://github.com/h2oai/wave/assets/63404062/44bcca3d-f8c7-4808-b759-d8b1b0f34b70">

Upon revisiting the application, it automatically restarts and resumes normal operation.
Could anyone kindly offer insights or suggestions on how to address this issue? Has anyone encountered a similar problem before? Any assistance would be greatly appreciated.

cc: @mturoci

**Comment by shamilh2o** [comment]
Experiencing same thing in one of my wave app as well. 
It uses wave version 1.1.1.

**Comment by mturoci** [comment, marked as answer]
The message seems like your wave app process has been killed by an OS. Try looking for system logs that could tell you more.

**Reply by nivzx** [reply]
This indeed was the issue. I was able to fix it after inspecting the system logs. Thanks!
